### PR TITLE
style entry page

### LINKED ISSUE #94: style entry page
At present, the entry page on mobile devices looks like
Screen Shot 2020-10-06 at 03 11 13

Align the title and area for nickname at the center of page i.e. align the jumbotron properly.

### PR DESCRIPTION
**Pull request by MonarthS**
Fixes #94

### CODE REVIEW

**Comment by MonarthS** [comment]
@LalitNM can you check this?

File: public/style.css [hunk 1]
```diff
@@ -13,6 +13,10 @@ li {
 }
 
 #user {
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
   font-family: 'Barrio', cursive;
   color: white;
   display: flex;
```
Comment (LalitNM) [inline public/style.css]: Remove all these rules and add bootstrap classes for them.

File: public/style.css [hunk 2]
```diff
@@ -52,7 +56,6 @@ li {
 
 @media screen and (max-width: 50em) {
   #user {
-    position: absolute;
```
Comment (LalitNM) [inline public/style.css:55]: remove this also.

**Review by LalitNM — CHANGES_REQUESTED** [review]
In short delete all rules from `style.css` for `#user`, `#bc`, add bootstrap classes for them, except the rules like `font-family` which you can't add using bootstrap. And listen to @kpbot.

And it seems like some of your commit are already there in `KamandPrompt: master`, use `git rebase upstream` for this.

File: public/style.css (hunk 1 above)
Comment (MonarthS) [reply public/style.css]: Okay, I'll check it.

Comment (MonarthS) [reply public/style.css]: Done.

File: public/style.css (hunk 2 above)
Comment (MonarthS) [reply public/style.css:55]: Already removed.

**Comment by LalitNM** [comment]
> In short delete all rules from style.css for #user, #bc, add bootstrap classes for them, except the rules like font-family which you can't add using bootstrap.

Do this also, and then I will review again

**Comment by MonarthS** [comment]
> > In short delete all rules from style.css for #user, #bc, add bootstrap classes for them, except the rules like font-family which you can't add using bootstrap.
> 
> Do this also, and then I will review again

**#user** contains only `font-family` & `display:flex`. We can't remove it and add bootstrap class `d-flex` as it overrides `.fadeOut()` or `.hide()`.

**#bc** contains only `font-size`.

**Comment by LalitNM** [comment]
You can further remove css rules, remove media query and add `display-1` class to #bc(also remove its font-size), remove font size of #userN, remove styling for #error_message. Do this and I will merge the PR.

**Comment by MonarthS** [comment]
Done changes.

File: public/style.css [hunk 3]
```diff
@@ -13,16 +13,7 @@ li {
...
-  width: 100%;
-  height: 100%;
-  justify-content: center;
-}
-
-#user>div {
-  margin: auto;
 }
 
 #user .home-label {
```
Comment (LalitNM) [inline public/style.css]: Can you remove this `#user` from here, I think it is useless.

**Review by LalitNM — CHANGES_REQUESTED** [review]
Looks good now, but line `39-42`, `15-17` can also be removed.

File: public/style.css (hunk 3 above)
Comment (MonarthS) [reply public/style.css]: `39-42` done. About `15-17` already told you [here](https://github.com/KamandPrompt/baat-cheet/pull/95#issuecomment-704129610). Also removed `#userN` style and added bootstrap classes.

**Review by LalitNM — APPROVED** [review]

**Comment by LalitNM** [comment]
Good work @MonarthS. Merging this PR :rocket:

**Pull request merged**